packages/@aws-cdk/aws-servicecatalog/test/portfolio.test.ts

```diff
@@ -457,7 +457,104 @@ describe('portfolio associations and product constraints', () => {
     expect(() => {
       portfolio.notifyOnStackEvents(product, topic);
     }).toThrowError(`Topic ${topic} is already subscribed to association`);
-  });
+  }),
+
+  test('set provisioning rule', () => {
```

Comment:
Looks like this is not the correct description - use the same name as in the docs ("CloudFormation parameters constraint").

Same comment for the other tests added in this PR.

Reply:
fixed